gateway/manual-tests/curl.sh

```diff
@@ -4,7 +4,7 @@
 # export ETH_ADDRESS='<put-your-public-key-here>'
 # export AVALANCHE_ADDRESS='<put-your-public-key-here>'
 
-# -k is --insecure, this disables certificate verfication and should only be
+# -k is --insecure, this disables certificate verification and should only be
 # used for local development and testing
 
```

Comment:
Hi @vic-en, as we discussed, we have unit and integration tests available for the controllers and the routes in the `gateway/integration-test` and `gateway/test` folders, where can test the full flows and also real tests with real money, if needed. These tests of ours bring a lot of flexibility. Anyway, I'll take a note to add some entries here in the future. In this case, I'll kindly ask you to leave this out of the PR for now, since we can test the system with those implementations there.